CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,56 @@
+# Changelog 2.3
+
+## 2.3.0 - 2025-02-19
+
+### Added
+
+- Add `Proxy` mode for Flow Aggregator to send flows directly without buffering or aggregation. ([#6920](https://github.com/antrea-io/antrea/pull/6920) [#6961](https://github.com/antrea-io/antrea/pull/6961), [@antoninbas])
+- Support version skew between Antrea Agent and Flow Aggregator to improve upgrade robustness. ([#6912](https://github.com/antrea-io/antrea/pull/6912), [@antoninbas])
```

Comment:
```suggestion
- Support version skew between antrea-agent and `FlowAggregator` to improve upgrade robustness. ([#6912](https://github.com/antrea-io/antrea/pull/6912), [@antoninbas])
```

Reply:
So do the following ones.

Reply:
I would prefer to `Antrea Agent` and `Flow Aggregator` in the release log. I feel antrea-agent is used when it's specific to the process.